spock-spring/boot-test/boot-test.gradle

```diff
@@ -23,6 +23,10 @@ dependencies {
   testCompile project(":spock-core")
   testCompile project(":spock-spring")
 
+  if (javaVersion >= 9) {
+    testCompile "javax.xml.bind:jaxb-api:2.3.0"
```

Comment:
Would it be better / more consistent to have the version or maybe even the java version check in one central place?

Reply:
Definitely :-). Improved.